aspnetcore/mvc/views/tag-helpers/builtin/resources/CacheTagHelper.md

```diff
@@ -0,0 +1,286 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+# Cache Tag Helper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+
+The  Cache Tag Helper provides the ability to dramatically improve the performance of your ASP.NET Core app by caching its content to the internal ASP.NET Core cache provider.
+
+A simple example that shows the Cache Tag Helper in action sets the current time in its content area. The Razor View Engine sets the default `expires-after` to twenty minutes, which is the default if you specify no additional attributes.
+
+Here is an example of the code you include in your `cshtml` page to achieve the default cache behavior described above.
+
+```html
+<Cache>@DateTime.Now<Cache>
+```
+
+If you repeatedly refresh the page that contains `CacheTagHelper`, the first time the current`DateTime`will be shown. Repeated refreshes of the page will continue to show the first `DateTime` shown until the cache expires (which is likely to be twenty minutes unless server memory pressure evicts the cache sooner).
```

Comment:
* Should `CacheTagHelper` here be "a Cache Tag Helper"?
* It needs a pair of spaces around the first `DateTime`
* The first sentence might need a touch ... If you "repeatedly" leads to "the first time" ... so it seems to mix periods of time. I think you'll need to have a short first sentence that states "the first time" bit. Allow the second sentence to address what happens when the page is refreshed multiple times thereafter.
* I think you can drop the parentheses and just introduce the "which" phrase with a comma.
* You can also totally drop the "to be" in favor of ", which is likely twenty minutes unless ..."